src/main/resources/validationrules/privatelaw/privatelaw-judicial-org-role-mapping.drl

```diff
@@ -505,4 +630,36 @@ then
       .authorisations($joh.getTicketCodes())
       .build());
       logMsg("Rule : privatelaw_magistrate_org_role");
-end;
\ No newline at end of file
+end;
+
+/*
+ * PRIVATELAW "magistrate" Org role mapping for v1_6.
+ */
+rule "v1_6privatelaw_magistrate_org_role"
+when
+  $f:  FeatureFlag(status && flagName == FeatureFlagEnum.PRIVATELAW_WA_1_6.getValue())
+  $joh: JudicialOfficeHolder(office in ("PRIVATELAW Magistrates-Voluntary"))
+then
+   Map<String,JsonNode> attribute = new HashMap<>();
+   attribute.put("contractType", JacksonUtils.convertObjectIntoJsonNode("Voluntary"));
+   attribute.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("PRIVATELAW"));
+   attribute.put("primaryLocation", JacksonUtils.convertObjectIntoJsonNode($joh.getPrimaryLocation()));
+   attribute.put("region", JacksonUtils.convertObjectIntoJsonNode($joh.getRegionId()));
+   attribute.put("workTypes", JacksonUtils.convertObjectIntoJsonNode("hearing_work, applications,routine_work"));
```

Comment:
inherited typo: can we remove extra space:
* was `hearing_work, applications,routine_work`
* update to `hearing_work,applications,routine_work`

Reply:
You may need to tweak unit test's expectedRoleNameWorkTypesMap to match.

Reply:
Resolved in 0b935b2a6808c7961ba5669080ae80d60314d886.